CHANGELOG-2.1.md

```diff
@@ -1,3 +1,13 @@
+# Changelog since v2.0.1
```

Comment:
this should be 2.1? and should 2.1.0's changes be included as well?

Reply:
It's a bit confusing, but 2.1.0's changelog is the "Changelog since v2.0.0" section.

What do you think of naming each section something like:
```
# v2.1.1 (Changelog since v2.1.0)

# v2.1.0 (Changelog since v2.0.0)
```

Reply:
Yeah, that's better.

Reply:
Done